x-pack/plugins/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -404,24 +404,47 @@ export class IndexTable extends Component {
   }
 
   renderBanners(extensionsService) {
-    const { allIndices = [], filterChanged } = this.props;
+    const { allIndices = [], filterChanged, performExtensionAction } = this.props;
     return extensionsService.banners.map((bannerExtension, i) => {
       const bannerData = bannerExtension(allIndices);
       if (!bannerData) {
         return null;
       }
 
-      const { type, title, message, filter, filterLabel } = bannerData;
+      const { type, title, message, filter, filterLabel, action } = bannerData;
 
       return (
         <Fragment key={`bannerExtension${i}`}>
           <EuiCallOut color={type} size="m" title={title}>
-            <EuiText>
-              {message}
-              {filter ? (
-                <EuiLink onClick={() => filterChanged(filter)}>{filterLabel}</EuiLink>
-              ) : null}
-            </EuiText>
+            {message && <p>{message}</p>}
```

Comment:
Can we wrap this with `<EuiText>`?

Reply:
Thanks for the review @ElenaStoeva !

`EuiCallout` already wrap children with `<EuiText>`, if we add another text wrap to message, the margin looks a little strange.

And in the eui demo example, the callout child elements are all p tags used directly.

![image](https://github.com/user-attachments/assets/7bbaca69-8afc-45c1-bfba-6b95a42f4664)